### DISCUSSION TITLE
ansible-blocks/

### DISCUSSION THREAD

**Discussion by giscus** [jvanderaa/jvanderaa.github.io#41] [Announcements]
# ansible-blocks/

One of the more interesting features that I have just come across within the Ansible automation world is that of the block. I find this very helpful for both error handling, and also grouping tasks into logical separation.
Ansible Official Link # Ansible Docs: Block
Blocks allow for logical grouping of tasks and in play error handling. Most of what you can apply to a single task can be applied at the block level, which also makes it much easier to set data or directives common to the tasks.

https://josh-v.com/ansible-blocks/

**Comment by jvanderaa** [comment]
Test on using comments on the blog with giscus.

**Reply by jvanderaa** [reply]
Replied online!

**Comment by jvanderaa** [comment]
Comment test within GitHub
